src/artemis/stores/mapgraph/core.cljs

```diff
@@ -40,16 +40,20 @@
 (defn create-store
   "Returns a new `MapGraphStore` for the given parameters:
 
-  - `:id-attrs`  A set of keys to normalize on. Defaults to `#{}`.
-  - `:entities`  A map of stored entities. Defaults to `{}`.
-  - `:cache-key` The default generic key for the store's cache. Defaults to
-                 `:artemis.stores.mapgrah.core/cache`."
+  - `:id-attrs`        A set of keys to normalize on. Defaults to `#{}`.
```

Comment:
✂️

Reply:
I'm going to go back and update all the markdown docs and docstrings in another PR